saleor/graphql/account/types.py

```diff
@@ -47,6 +48,7 @@ class User(CountableDjangoObjectType):
         PrefetchingConnectionField(
             Address, description='List of all user\'s addresses.'),
         model_field='addresses')
+    note = graphene.String(description='')
```

Comment:
We could add the description here, I think a suitable one would be "A note about the customer".